examples/remix/backend/s3Client.server.ts

```diff
@@ -0,0 +1,3 @@
+import { S3Client, ListBucketsCommandInput, ListObjectsV2Command, ListObjectsV2CommandInput, ListObjectsV2CommandOutput, S3ClientConfig, ListBucketsCommand } from '@aws-sdk/client-s3';
+
+export const s3Client = new S3Client({ region: 'eu-central-1', endpoint: 'http://localhost:4566' });
```

Comment:
The endpoint right now, which server is that?

And when deployed, which server should it be then?

Should we not instead use environment variables to see whether we are in DEV or some other environment and depending on that we change the endpoint's url?